examples/src/walkable-path.ts

```diff
@@ -0,0 +1,151 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from 'three';
+import CameraControls from 'camera-controls';
+import * as reveal from '@cognite/reveal';
+import { CogniteClient } from '@cognite/sdk';
+import { toThreeVector3, CadNode } from '@cognite/reveal/threejs';
+import { vec3 } from 'gl-matrix';
+import { SectorModelTransformation } from '@cognite/reveal/models/cad/types';
+
+CameraControls.install({ THREE });
+
+interface TransitPathResponse {
+  items: [
+    {
+      segments: [
+        {
+          path: [
+            {
+              x: number;
+              y: number;
+              z: number;
+            }
+          ];
+        }
+      ];
+    }
+  ];
+}
+
+interface TransitPathRequest {
+  modelId: number;
+  items: [
+    {
+      from: { x: number; y: number; z: number } | { nodeId: number };
+      to: { x: number; y: number; z: number } | { nodeId: number };
+      movingObjectSize: { diameter: number; height: number };
+    }
+  ];
+}
+
+// Should find a way to add this to maybe cognite-sdk-js later?
+async function getTransitPath(client: CogniteClient, transitRequest: TransitPathRequest) {
+  const project = '3ddemo';
+  const url = `https://api.cognitedata.com/api/playground/projects/${project}/3d/pathfinder/transit`;
+  const response = await client.post<TransitPathResponse>(url, {
+    data: transitRequest
+  });
+  return response;
+}
```

Comment:
Move to file in viewer/. For outputs, Im creating a 'CogniteClientExtensions'-class that the plan is to add hidden/non-production endpoints to. See https://github.com/cognitedata/reveal/pull/134/commits/a729c56df4b97ed941cbaed5b10ec5935d89d601#diff-ef96b80e3bb59fdb8b6ce79524587ee0 (this is by no means perfect yet).

Reply:
I moved the client to a separate function but kept it in the example. For now I think that is useful, and I would like to revisit this once we have gone through our architecture talk in the next sprint.